tests/foreman/api/test_docker.py

```diff
@@ -1083,146 +1083,77 @@ def setUpClass(cls):
             url=EXTERNAL_DOCKER_URL,
         ).create()
 
-    def setUp(self):
-        """Generate default ``DockerHubContainer`` keyword arguments.
-
-        Generate a dict of arguments that can be used when instantiating a
-        ``DockerHubContainer`` and save it as ``self.dhc_kwargs``.
-
-        """
-        self.container_name = gen_string('alphanumeric')
-        self.dhc_kwargs = {
-            'command': 'top',
-            'name': self.container_name,
-            'organization': [self.org],
-        }
-
-    def test_create_container_local_compute_resource(self):
-        """@Test: Create a container in a local compute resource
-
-        @Feature: Docker
-
-        @Assert: The docker container is created in the local compute resource
-
-        """
-        container = entities.DockerHubContainer(
-            compute_resource=self.cr_internal,
-            **self.dhc_kwargs
-        ).create()
-        self.assertEqual(
-            container.compute_resource.read().name,
-            self.cr_internal.name,
-        )
-
-    def test_create_container_external_compute_resource(self):
-        """@Test: Create a container in an external compute resource
+    def test_create_container_compute_resource(self):
```

Comment:
I saw that approach before, but didn't use it as we have corresponding references in Polarion. I think that we need to  keep in mind such fact. If you like to use single names for both external and local entities, please perform update in the tool and write here, so I can check that fact. Thanks